src/wpt/BUILD.bazel

```diff
@@ -49,7 +49,6 @@ wpt_test(
 
 wpt_test(
     name = "urlpattern-standard",
-    compat_flags = ["urlpattern_standard"],
```

Comment:
Why is this needed?

Reply:
The test currently fails with "This compat flag is default starting 2025-05-01 so no longer needed". (this should probably be a warning instead of an error though?)